civicase.php

```diff
@@ -647,6 +655,15 @@ function civicase_civicrm_permission_check($permission, &$granted) {
  * Implements hook_civicrm_preProcess().
  */
 function civicase_civicrm_preProcess($formName, &$form) {
+  $hooks = [
+    new CRM_Civicase_Hook_PreProcess_CaseCategoryCustomFieldsAdder(),
+    new CRM_Civicase_Hook_PreProcess_CaseCategoryCustomFieldsSetDefaultValues(),
+  ];
+
+  foreach ($hooks as $hook) {
+    $hook->run($formName, $form);
+  }
+
   if ($formName == 'CRM_Admin_Form_Setting_Case') {
```

Comment:
Perhaps add TODO for moving CRM_Admin_Form_Setting_Case form on preProcess hook into class in the future.